.github/workflows/ci.yml

```diff
@@ -47,9 +47,16 @@ jobs:
       cedar_policy_ref: ${{ needs.get-branch-name.outputs.branch_name }}
 
   test_docker_build:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest-large
+    runs-on: ${{ matrix.os }}
     name: Test Docker build
-    runs-on: ubuntu-latest
     steps:
+      - name: Set up Docker
+        uses: crazy-max/ghaction-setup-docker@v3
```

Comment:
Is there any potential issue with trusting  this `crazy-max`?

Reply:
Good question. It should be fine since we're not using any private keys?

Reply:
Let me try to get rid of it.

Reply:
I managed to do it but build on mac is taking a long time. Optimizing.